internal/serverinstall/k8s.go

```diff
@@ -1690,6 +1733,31 @@ func (i *K8sInstaller) initServiceAccount(
 		return err
 	}
 
+	cr, crb, err := newServiceAccountClusterRoleWithBinding(i.config)
+	if err != nil {
+		return status.Errorf(codes.Internal, "Failed to get definition for runner service account's cluster role and binding: %s", err)
+	}
+	if cr != nil {
+		crClient := clientset.RbacV1().ClusterRoles()
+		_, err = crClient.Get(ctx, cr.Name, metav1.GetOptions{})
+		if err != nil && !errors.IsNotFound(err) {
+			return err
+		}
+		if _, err := crClient.Create(ctx, cr, metav1.CreateOptions{}); err != nil {
+			return status.Errorf(codes.Internal, "Failed to create cluster role %s: %s", cr.Name, err)
+		}
+	}
+	if crb != nil {
+		crbClient := clientset.RbacV1().ClusterRoleBindings()
+		_, err = crbClient.Get(ctx, crb.Name, metav1.GetOptions{})
+		if err != nil && !errors.IsNotFound(err) {
+			return err
```

Comment:
Should we wrap the 2 `IsNotFound` errors in `status.Errorf` here, to be consistent with the other error scenarios from the `newServiceAccountClusterRoleWithBinding`? Or is directly returning the error more appropriate?

Reply:
Oh nice catch, thank you! That check was a late addition copy-paste, and I missed customizing the error.